server/views/partials/summary-card-with-steps.njk

```diff
@@ -0,0 +1,60 @@
+{% from "../components/summary-card/summary-card.njk" import summaryCard %}
+
+{% macro summaryListWithSteps (goal) %}
+    {% call summaryCard({
+        title: {
+            text:  goal.title
+        },
+        actions: {
+          items: [
+            {
+              href: "/edit-goal",
+              text: "Change goal",
+              visuallyHiddenText: "Change goal"
+            },
+            {
+              href: "/add-change-steps",
+              text: "Add or change steps",
+              visuallyHiddenText: "Add or change steps"
+            },
+            {
+              href: "/remove-goal",
+              text: "Remove goal",
+              visuallyHiddenText: "Remove goal"
+            }
+          ]
+        }
+    }) %}
+
+    <p class="govuk-body">Aim to achieve {{ goal.targetDate | formatSimpleDate }}</p>
```

Comment:
This doesn't match the design attached to the ticket

Reply:
updated